src/runtime-rs/crates/resource/src/share_fs/share_virtio_fs_inline.rs

```diff
@@ -22,10 +22,8 @@ use super::{
 };
 
 lazy_static! {
-    pub(crate) static ref SHARED_DIR_VIRTIO_FS_OPTIONS: Vec::<String> = vec![
-        String::from("default_permissions,allow_other,rootmode=040000,user_id=0,group_id=0"),
-        String::from("nodev"),
-    ];
+    pub(crate) static ref SHARED_DIR_VIRTIO_FS_OPTIONS: Vec::<String> =
+        vec![String::from("nodev"),];
```

Comment:
I do wonder if this vec couldn't be created on demand from a `const &str` value of `"nodev"`.

Reply:
Thanks for your review. We need to use `Vec<String>`, when using `Vec<&str>` also need to convert to `Vec<String>`.